tests/dummy/config/environment.js

```diff
@@ -0,0 +1,46 @@
+module.exports = function (environment) {
+  var ENV = {
+    modulePrefix: 'dummy',
+    podModulePrefix: 'dummy/pods',
+    environment: environment,
+    baseURL: '/ember-frost-demo-components',
+    locationType: 'auto',
+    EmberENV: {
+      FEATURES: {
+        // Here you can enable experimental features on an ember canary build
+        // e.g. 'with-controller': true
+      }
+    },
+
+    APP: {
+      // Here you can pass flags/options to your application instance
+      // when it is created
+    }
+  }
+
+  if (environment === 'development') {
+    // ENV.APP.LOG_RESOLVER = true
+    // ENV.APP.LOG_ACTIVE_GENERATION = true
+    // ENV.APP.LOG_TRANSITIONS = true
+    // ENV.APP.LOG_TRANSITIONS_INTERNAL = true
+    // ENV.APP.LOG_VIEW_LOOKUPS = true
+  }
+
+  if (environment === 'test') {
+    // Testem prefers this...
+    ENV.baseURL = '/'
+    ENV.locationType = 'none'
+
+    // keep test console output quieter
+    ENV.APP.LOG_ACTIVE_GENERATION = false
+    ENV.APP.LOG_VIEW_LOOKUPS = false
+
+    ENV.APP.rootElement = '#ember-testing'
+  }
+
+  if (environment === 'production') {
+
+  }
+
+  return ENV
+}
```

Comment:
I know this file is auto-generated but it would look a bit cleaner if it were:
```javascript
module.exports = function (environment) {
  var ENV = {
    APP: {},
    baseURL: '/ember-frost-demo-components',
    EmberENV: {
      FEATURES: {}
    },
    environment: environment,
     locationType: 'auto',
    modulePrefix: 'dummy',
    podModulePrefix: 'dummy/pods'
  }

  switch (environment) {
    case 'test':
      Ember.assign(ENV, {
        baseURL: '/',
        locationType: 'none'
      })

      Ember.assign(ENV.APP, {
        APP.LOG_ACTIVE_GENERATION: false,
        APP.LOG_VIEW_LOOKUPS: false,
        rootElement: '#ember-testing'
      })

      break
  }

  return ENV
}
```